### ISSUE TITLE
Bug: Tooltip for LineChart shows incorrect labels when there is a dual axis

### ISSUE THREAD

**Issue by nstuyvesant** [carbon-design-system/carbon-charts#1085]
#### I am submitting a...

- [ ] Feature request
- [ ] Design defect
- [x] Source code defect
- [ ] Demo/documentation defect
- [ ] Other

#### charts version:
0.41.91

### Issue description
Tooltips for line charts 

#### Steps to reproduce the issue

1. Open [Storybook demo of Line + Line (dual axes)](https://carbon-design-system.github.io/carbon-charts/svelte/?path=/story/simple-charts-line--line-line-dual-axes)
2. Hover over a data point for the Rainfall Group.

#### Current behavior
Total label is "Temperature (°C)". It does not appear to be an issue of incorrect configuration with the options. I am having the same issue with my code that's similar.

#### Expected behavior
Total label should be "Rainfall (mm)"

### Screenshot or recording
See steps to reproduce the issue

Guessing this issue may have been an oversight where dual axis graphs are concerned.

**Comment by theiliad** [comment]
Hi, I'm having trouble understanding where the issue is, could you please explain in more details?

Including screenshots would also be great

**Comment by nstuyvesant** [comment]
In this screenshot, you see the label next to the value is not in the units corresponding to that dataset's units: <img width="232" alt="image" src="https://user-images.githubusercontent.com/5770938/126559103-8cf0a975-f8fd-4d08-a83a-5d433c4f61df.png">

**Comment by nstuyvesant** [comment]
@theiliad - Did this explanation help or do you still have questions?

**Comment by nstuyvesant** [comment]
Here's the HTML for the auto-generated tooltip I captured using a customHTML function...
```html
  <ul class='multi-tooltip'>
    <li>
      <div class="datapoint-tooltip ">
        <div class="label">
          <p>Date</p>
        </div>
        <p class="value"/>Jun 30, 2021</p>
      </div>
    </li>
    <li>
      <div class="datapoint-tooltip ">
        <div class="label">
          <p>Attendees</p>
        </div>
        <p class="value"/>2,625</p>
      </div>
    </li>
    <li>
      <div class="datapoint-tooltip ">
        <a class="tooltip-color tooltip-2-1-2"></a>
        <div class="label">
          <p>Group</p>
        </div>
        <p class="value"/>Revenue</p>
      </div>
    </li>
  </ul>
```

Here's what it should have contained...
```html
  <ul class='multi-tooltip'>
    <li>
      <div class="datapoint-tooltip ">
        <div class="label">
          <p>Date</p>
        </div>
        <p class="value"/>Jun 30, 2021</p>
      </div>
    </li>
    <li>
      <div class="datapoint-tooltip ">
        <div class="label">
          <p>Revenue</p>
        </div>
        <p class="value"/>2,625</p>
      </div>
    </li>
    <li>
      <div class="datapoint-tooltip ">
        <a class="tooltip-color tooltip-2-1-2"></a>
        <div class="label">
          <p>Group</p>
        </div>
        <p class="value"/>Revenue</p>
      </div>
    </li>
  </ul>
```

Note the second div with class="datapoint-tooltip". The paragraph should have contained "Revenue" from the axes.left.title but it duplicated the axes.right.title of "Attendees".

**Comment by nstuyvesant** [comment]
To workaround the issue, I added a customHTML function to rewrite the tooltip contents:
```
<LineChart data={$dashboardData.attendanceRevenue} options={{
      tooltip: {
        customHTML: (data, defaultHTML) => { // workaround for bug in carbon charts
          let result = defaultHTML
          if (data.length == 1) {
            result = `
              <ul class="multi-tooltip">
                <li>
                  <div class="datapoint-tooltip">
                    <div class="label">
                      <p>Date</p>
                    </div>
                    <p class="value">${data[0].date.toLocaleDateString('en-US', {year: 'numeric', month: 'short', day: 'numeric'})}</p>
                  </div>
                </li>
                <li>
                  <div class="datapoint-tooltip">
                    <a class="tooltip-color tooltip-2-1-${data[0].group == 'Revenue' ? '2': '1'}"></a>
                    <div class="label">
                      <p>${data[0].group}</p>
                    </div>
                    <p class="value">${data[0].group == 'Revenue' ? data[0].revenue.toLocaleString('en-US', { style: 'currency', currency: 'USD', maximumFractionDigits: 0 }): data[0].attendees.toLocaleString('en-US')}</p>
                  </div>
                </li>
              </ul>`
          }
          return result
        }
      },
      axes: {
        bottom: {
          mapsTo: 'date',
          title: 'Date',
          scaleType: 'time'
        },
        left: {
          mapsTo: 'revenue',
          title: 'Revenue',
          scaleType: 'linear',
          main: true,
          correspondingDatasets: [
            'Revenue'
          ]
        },
        right: {
          mapsTo: 'attendees',
          title: 'Attendees',
          scaleType: 'linear',
          correspondingDatasets: [
            'Attendees'
          ]
        }
      },
      curve: 'curveMonotoneX',
      height: '500px'
    }}/>
```

**Comment by nstuyvesant** [comment]
Still a problem in 0.50.4

**Comment by nstuyvesant** [comment]
This is not related to #1188 as the fix for it in @carbon/charts@0.53.1 did not resolve the issue.

In this screenshot, the value next to the number should say "Revenue", not "Attendees".
<img width="123" alt="image" src="https://user-images.githubusercontent.com/5770938/144473481-dc74b171-2f45-4836-a0a5-16537c48cc65.png">
